docs/pages/example/create-color-ramp-for-building-layer.html

```diff
@@ -0,0 +1,52 @@
+<style>
+    #zoom {
```

Comment:
Let's center the button and style it like those in https:www.mapbox.com/mapbox-gl-js/example/language-switch/ and https://www.mapbox.com/mapbox-gl-js/example/flyto/ for consistency.